DxDispatch/src/model/OnnxParsers.cpp

```diff
@@ -0,0 +1,173 @@
+#include "pch.h"
+#include "OnnxParsers.h"
+#include "DirectMLX.h"
+#include "dml_provider_factory.h"
+
+std::string OnnxParsers::GetTensorName(size_t index, Ort::Session const& session, bool isInput)
+{
+    Ort::AllocatorWithDefaultOptions allocator;
+    char* name = isInput ? session.GetInputName(index, allocator) : session.GetOutputName(index, allocator);
+    std::string returnName(name);
+    allocator.Free(name); // Don't leak memory.
+    return returnName;
+}
+
+bool OnnxParsers::IsSupportedOnnxTensorElementDataType(ONNXTensorElementDataType dataType)
+{
+    switch (dataType)
+    {
+    case ONNX_TENSOR_ELEMENT_DATA_TYPE_UNDEFINED:   return true;
```

Comment:
>true [](http://example.com/codeflow?start=59&length=4)

I fixed this to false. Sorry.

[https://github.com/fdwr/OnnxRuntimeDirectMLEPSample/blob/e2f7123272d69651a046fd69442febb958aaeb99/main.cpp#L79](https://github.com/fdwr/OnnxRuntimeDirectMLEPSample/blob/e2f7123272d69651a046fd69442febb958aaeb99/main.cpp#L79)